scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigtable/syntax/ScioContextSyntax.scala

```diff
@@ -239,128 +146,44 @@ final class ScioContextOps(private val self: ScioContext) extends AnyVal {
    *   A map of tables and column families. Keys are table names. Values are a list of column family
    *   names.
    */
-  def ensureTables(
-    projectId: String,
-    instanceId: String,
-    tablesAndColumnFamilies: Map[String, Iterable[String]],
-    createDisposition: TableAdmin.CreateDisposition
-  ): Unit =
-    if (!self.isTest) {
-      val bigtableOptions = BigtableOptions
-        .builder()
-        .setProjectId(projectId)
-        .setInstanceId(instanceId)
-        .build
-      TableAdmin.ensureTables(bigtableOptions, tablesAndColumnFamilies, createDisposition)
-    }
-
-  def ensureTables(
+  def ensureTable(
     projectId: String,
     instanceId: String,
-    tablesAndColumnFamilies: Map[String, Iterable[String]]
-  ): Unit = ensureTables(
-    projectId,
-    instanceId,
-    tablesAndColumnFamilies,
-    TableAdmin.CreateDisposition.default
-  )
-
-  /**
-   * Ensure that tables and column families exist. Checks for existence of tables or creates them if
-   * they do not exist. Also checks for existence of column families within each table and creates
-   * them if they do not exist.
-   *
-   * @param tablesAndColumnFamilies
-   *   A map of tables and column families. Keys are table names. Values are a list of column family
-   *   names.
-   */
-  def ensureTables(
-    bigtableOptions: BigtableOptions,
-    tablesAndColumnFamilies: Map[String, Iterable[String]],
-    createDisposition: TableAdmin.CreateDisposition
+    tableId: String,
```

Comment:
As underlying client is cached, simplify the API an not worry if this is called multiple times.